test/mavsdk_tests/process_helper.py

```diff
@@ -176,15 +176,25 @@ def __init__(self, workspace_dir: str, log_dir: str,
     def clear_rootfs(self) -> None:
         rootfs_path = self.cwd
         if self.verbose:
-            print("Deleting rootfs: {}".format(rootfs_path))
+            print("Clearing rootfs (except logs): {}".format(rootfs_path))
```

Comment:
Maybe we could add something that makes it clear which log corresponds to which test?

I also wouldn't be opposed to throwing them all at https://logs.px4.io/ with metadata populated so we know exactly which test/build/PR it corresponds to.